t/core/request.t

```diff
@@ -439,3 +439,47 @@ while ($i <= 101) {
 $s
 --- response_body
 101
+
+
+
+=== TEST 15: add header
+--- config
+    location /t {
+        content_by_lua_block {
+            local core = require("apisix.core")
+            ngx.ctx.api_ctx = {}
+            local ctx = ngx.ctx.api_ctx
+            core.request.add_header(ctx, "test_header", "test")
+            local h = core.request.header(ctx, "test_header")
+            ngx.say(h)
+            core.request.add_header(ctx, "test_header", "t2")
+            local h2 = core.request.header(ctx, "test_header")
+            ngx.say("[" .. table.concat(h2, ", ") .. "]")
```

Comment:
is it better to use `core.json.encode` instead?

Reply:
done